### PR TITLE
Updated to liberty-maven-plugin-2.0-SNAPSHOT

### PR DESCRIPTION
**Pull request by josephca**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -293,9 +204,6 @@
 							<goal>integration-test</goal>
 						</goals>
 						<configuration>
-							<includes>
```
Comment (patricktiu) [inline pom.xml]: I did not find any unit test or IT test code in this project.  Maybe we can get rid of the maven-failsafe-plugin

Comment (patricktiu) [inline pom.xml]: We create pull request against your own fork master for now.  After we have released liberty-maven-plugin 2.0, we will create another PR from your fork master to the real master.

Comment (patricktiu) [inline pom.xml]: Please make sure your project can be imported to WDT and Run as -> Run on Server.

Comment (patricktiu) [inline pom.xml]: Instruction for **Running with Maven running** in README.md 

This project can be built with Apache Maven. The project uses Liberty Maven Plug-in to automatically download and install Liberty ~~profile~~ `with Java EE7 Web Profile `  runtime from the ~~Liberty repository~~ `Maven Central`. Liberty Maven Plug-in is also used to create, configure, and run the application on the Liberty server.

Comment (patricktiu) [inline pom.xml]: Instruction in README.md to start server in Eclipse should be

4. Right-click the project and select Run As -> Run on Server.

**Review by patricktiu — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (josephca) [reply pom.xml]: Sure, this will be closed for now and will create new PR.

**Comment by josephca** [comment]
Closing it for now and will re-open once maven-plugin-2.0 is released.

**Pull request closed without merging**